core/src/organization/member-restriction/member-restriction.types.ts

```diff
@@ -1,6 +1,8 @@
 import {Field, ObjectType} from "@nestjs/graphql";
 
-import {MemberRestriction} from "$models";
+// For some reason this doesn't exist on $models
+// eslint-disable-next-line no-restricted-imports
+import {MemberRestriction} from "../../database/organization/member_restriction/member_restriction.model";
```

Comment:
This is weird 🤔 Try to fix?

Reply:
I have no clue what the issue is. Importing from $models and logging it just does undefined, but it’s exported everywhere.

Reply:
And is only an issue in this file.

Reply:
If its `undefined`, its likely a circular reference caused by a barrel file somewhere